docs/output.md

```diff
@@ -6,554 +6,671 @@ This document describes the output produced by the pipeline. Most of the plots a
+
+**Output directory: `{outdir}/preprocessing/csv`**
+
+- `mapped.csv`
+  - if `--save_bam_mapped`
+  - CSV containing an entry for each sample with the columns `patient,sample,sex,status,bam,bai`
+- `markduplicates_no_table.csv`
+  - CSV containing an entry for each sample with the columns `patient,sample,sex,status,cram,crai`
+- `markduplicates.csv`
+  - CSV containing an entry for each sample with the columns `patient,sample,sex,status,cram,crai,table`
+- `recalibrated.csv`
+  - CSV containing an entry for each sample with the columns`patient,sample,sex,status,cram,crai`
+- `variantcalled.csv`
+  - CSV containing an entry for each sample with the columns `patient,sample,vcf`
+  </details>
 
-- `HaplotypeCaller_[SAMPLE].vcf.gz` and `HaplotypeCaller_[SAMPLE].vcf.gz.tbi`
-  - `VCF` with Tabix index
+## Variant Calling
 
-For further reading and documentation see the [HaplotypeCaller manual](https://gatk.broadinstitute.org/hc/en-us/articles/360042913231-HaplotypeCaller).
+The results regarding variant calling are collected in `{outdir}/variantcalling/`.
+If some results from a variant caller do not appear here, please check out the `--tools` section in the parameter [documentation](https://nf-co.re/sarek/3.0.0/parameters).
 
-#### GATK GenotypeGVCFs
+(Recalibrated) CRAM files can used as an input to start the variant calling.
 
-[GATK GenotypeGVCFs](https://gatk.broadinstitute.org/hc/en-us/articles/360042914991-GenotypeGVCFs) performs joint genotyping on one or more samples pre-called with HaplotypeCaller.
+### SNVs and small indels
 
-Germline calls are provided for all samples, to enable comparison of both, tumor and normal, for possible mixup.
+For single nucleotide variants (SNVs) and small indels, multiple tools are available for normal, tumor-only, and tumor-normal paired data. For a list of the appropriate tool(s) for the data and sequencing type at hand, please check [here](usage.md#which-tool).
 
-For all samples:
+#### DeepVariant
 
-**Output directory: `results/VariantCalling/[SAMPLE]/HaplotypeCallerGVCF`**
+[DeepVariant](https://github.com/google/deepvariant) is a deep learning-based variant caller that takes aligned reads, produces pileup image tensors from them, classifies each tensor using a convolutional neural network, and finally reports the results in a standard VCF or gVCF file. For further documentation take a look [here](https://github.com/google/deepvariant/tree/r1.4/docs).
 
-- `HaplotypeCaller_[SAMPLE].g.vcf.gz` and `HaplotypeCaller_[SAMPLE].g.vcf.gz.tbi`
-  - `VCF` with Tabix index
+<details markdown="1">
+<summary>Output files for normal samples</summary>
 
-For further reading and documentation see the [GenotypeGVCFs manual](https://gatk.broadinstitute.org/hc/en-us/articles/360042914991-GenotypeGVCFs).
+**Output directory: `{outdir}/variantcalling/<sample>/deepvariant`**
 
-#### GATK Mutect2
+- `<sample>.deepvariant.vcf.gz` and `<sample>.deepvariant.vcf.gz.tbi`
+  - VCF with tabix index
```

Comment:
```suggestion
  - `VCF` with tabix index
```

Reply:
on anders suggestion I removed this quotes everywhere because we were using too much code style text